sound/soc/sof/pm.c

```diff
@@ -317,8 +322,11 @@ int snd_sof_suspend(struct device *dev)
 			return ret;
 		}
 
-		/* platform-specific suspend to D0i3 */
-		return snd_sof_dsp_suspend(sdev);
+		/*
+		 * platform-specific suspend to D0i3
+		 * FIXME: Remove hard-coded state
+		 */
+		return snd_sof_dsp_suspend(sdev, SOF_DSP_D0I3);
```

Comment:
Oh man, this patch is out of order on github. the "previous" patch doesn't have any of this